packages/design-system/src/components/Dropdown/Dropdown.jsx

```diff
@@ -33,10 +33,14 @@ export class Dropdown extends React.PureComponent {
         {...containerProps}
         component="div"
         labelComponent="label"
-        render={({ id, setRef }) => (
+        render={({ id, errorId, setRef }) => (
           <Select
             {...inputOnlyProps}
             {...{ id, setRef }}
+            // Link input to bottom placed error message
+            aria-describedBy={
```

Comment:
```suggestion
            aria-describedby={
```

Reply:
Did you end up committing this change? @nichia did it work ok?

Reply:
Not sure why, but when I tried the using `aria-describedby`, it doesn't show up on the html. So I did not commit this change and we'll just see a warning message of `aria-describedBy`